unity-renderer/Assets/DCLPlugins/ECS7/InternalECSComponents/Interfaces/Extensions/InternalTexturizableExtensions.cs

```diff
@@ -31,7 +31,7 @@ public static class InternalTexturizableExtensions
             if (renderers == null)
                 return;
 
-            var model = texurizableInternalComponent.GetFor(scene, entity)?.model ?? new InternalTexturizable();
+            var model = texurizableInternalComponent.GetFor(scene, entity)?.model ?? new InternalTexturizable(new List<Renderer>());
```

Comment:
here as well - is it possible to cache, clear and re-use instead of creating new instances?